client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for CodeEvalResultObject domain objects.
+ */
+
+describe('CodeEvalResultObjectFactory', function() {
+  var CodeEvalResultObjectFactory;
+
+  beforeEach(module('tie'));
+  beforeEach(inject(function($injector) {
+    CodeEvalResultObjectFactory = $injector.get(
+      'CodeEvalResultObjectFactory');
+  }));
+
+  describe('getOutput', function() {
+    it('should correctly get output', function() {
+      var codeEvalResult = CodeEvalResultObjectFactory.create("code", "output",
```

Comment:
Might want to set the various strings to vars as well so you can access them later instead of using hardcoded strings everywhere.